api/restHandler/app/DeploymentPipelineRestHandler.go

```diff
@@ -438,6 +438,15 @@ func (handler PipelineConfigRestHandlerImpl) GetEnvConfigOverride(w http.Respons
 		common.WriteJsonResp(w, err, nil, http.StatusInternalServerError)
 		return
 	}
+
+	schema, readme, err := handler.chartService.GetSchemaAndReadmeForDefaultTemplate(chartRefId)
+	if err != nil {
+		handler.Logger.Errorw("err in getting schema and readme, GetDeploymentTemplate", "err", err, "appId", appId, "chartRefId", chartRefId)
```

Comment:
this should be GetEnvConfigOverride instead of GetDeploymentTemplate in log message?